tests/financial/test_nasdaq.py

```
@@ -0,0 +1,376 @@
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.constants import HALF_DAY
+from holidays.financial.nasdaq import NASDAQ
+from tests.common import CommonFinancialTests
+
+
+class TestNASDAQ(CommonFinancialTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.15.14)

[warning] Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)